config/routes/practitioner.js

```diff
@@ -0,0 +1,26 @@
+module.exports.routes = {
+  /**
+   * Practitioner ROUTES
+   */
+  'GET /en/practitioner': {
+    name: 'practitioner',
+    controller: 'PractitionerController',
+    action: 'index',
+    lang: 'en',
+    i18n: {
+      en: '/en/practitioner',
+      fr: '/fr/practitioner'
+    }
+  },
+
+  'POST /en/practitioner': {
+    name: 'practitioner.store',
+    controller: 'PractitionerController',
+    action: 'store',
+    lang: 'en',
+    i18n: {
+      en: '/en/practitioner',
+      fr: '/fr/practitioner'
+    }
+  },
+}
```

Comment:
Fix linting errors add a `;` and a new line

Reply:
fixed